data/player/Admin.json

```diff
@@ -1 +1 @@
-{"attributes":{"maxHealth":100,"health":100,"maxMana":200,"mana":200,"maxEnergy":200,"energy":200,"strength":20,"intelligence":20,"wisdom":20,"dexterity":20,"constitution":20},"experience":0,"inventory":[],"level":1,"name":"Admin","prefs":[],"questData":{"completed":[],"active":[["limbo:1",{"state":{"count":0}}]]},"room":"limbo:1","equipment":{}}
\ No newline at end of file
+{"attributes":{"health":100,"mana":200,"energy":200,"strength":20,"intelligence":20,"wisdom":20,"dexterity":20,"constitution":20},"experience":0,"inventory":[],"level":1,"name":"Admin","prefs":[],"questData":{"completed":[],"active":[["limbo:1",{"state":{"count":0}}]]},"room":"limbo:1","equipment":{}}
```

Comment:
Ah, that's correct. However, the Attributes class will still handle Numbers in the constructor since it has to do so to handle NPC definitions and the default Player attributes as specified in ranvier.json. So, this won't blow up either way, but I could go ahead and change it to match the serialized format.